docs/modules/rust-guide/pages/multiply-dependency.adoc

```diff
@@ -248,7 +248,7 @@ service : {
 
 == Start the local network
 
-Before you can build the project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a sub-network that you can access.
+Before you can build the project, you need to connect to the {IC} network either running locally in your development environment or the decentralized {IC} mainnet.
```

Comment:
I really like the term _the IC blockchain network_.

Can't you use this here as well? The fact that it has a define article indicates that there is only one and hence the "mainnet".

I'm not sure that the term "mainnet" adds much in general use, as the IC blockchain network doesn't have any testnets.

Reply:
The discussion seems to favor "Internet Computer blockchain" so I updated to that.  I converted the phrase to an attribute and added the modifier "mainnet" only when required to differentiate from the local IC.